tests/integration/targets/backup_selection/tasks/main.yml

```
@@ -0,0 +1,275 @@
+---
+- module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+
+  block:
+    - name: Create an IAM Role
+      community.aws.iam_role:
+        name: "{{ backup_iam_role_name }}"
+        assume_role_policy_document: '{{ lookup("file", "backup-policy.json") }}'
+        create_instance_profile: no
+        description: "Ansible AWS Backup Role"
+        managed_policy:
+          - "arn:aws:iam::aws:policy/service-role/AWSBackupServiceRolePolicyForBackup"
+      register: iam_role
+
+    # Wait for the role to be created
+    - pause:
+        seconds: 20
```

If we use wait: True we should be able to drop this to 5.

```suggestion
        seconds: 5
```